packages/embeds/embed-core/playwright/lib/testUtils.ts

```diff
@@ -84,7 +84,7 @@ async function selectFirstAvailableTimeSlotNextMonth(frame: Frame, page: Page) {
 
   // Waiting for full month increment
   await frame.waitForTimeout(1000);
-  expect(await page.screenshot()).toMatchSnapshot("availability-page-2.png");
```

Comment:
Commented these snapshots, they were never compared. But I plan to work on logic to compare them.